R/Random.R

```diff
@@ -0,0 +1,12 @@
+#Reconstruct a network from a random matrix, not taking time series into account
+#@export
```

Comment:
So comments need to be of the form `#'` to be parsed.  An export would look like `#' @export`.